juju/unit.py

```diff
@@ -138,11 +138,11 @@ def resolved(self, retry=False):
             timeout = int(timeout * 1000000000)
 
         res = await action.Run(
-            [],
-            command,
-            [],
-            timeout,
-            [self.name],
+            applications=[],
+            commands=command,
+            machines=[],
+            timeout=timeout,
+            units=[self.name],
```

Comment:
I like passing in the named parameters a lot. I think its much easier to understand the request than lots of '[], command, [], ' (what is the 3rd argument mean)